src/thor/isochrone.cc

```diff
@@ -40,7 +40,6 @@ uint32_t GetOperatorId(const GraphTile* tile,
 namespace valhalla {
 namespace thor {
 
-constexpr uint32_t kBucketCount = 20000;
```

Comment:
this was already in the pathalgorithm header which is now included to get the TimeInfo struct